src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -30,8 +30,8 @@
               <!-- Person's Name -->
               <template v-if="isPerson">
                 <label class="sub-header">Person's Name</label>
-                <p v-if="isProprietor" class="info-text mb-0">
-                  If the proprietor has changed their legal name, enter their new legal name.
+                <p v-if="isProprietor || isPartner" class="info-text mb-0">
+                  If the {{ orgPersonLabel2 }} has changed their legal name, enter their new legal name.
```

Comment:
I had to bounce around in the code quite a bit to get my head around orgPersonLabel, orgPersonLabel2 and orgPersonLabel3.

I recommend you simplify this by simply coding the text using a ternary like this:

https://github.com/bcgov/business-create-ui/blob/a4b719c46f0ce48239d79672f2e005f68a270b55/src/components/Registration/RegAddEditOrgPerson.vue#L158